elements/pl-code/pl-code.py

```diff
@@ -20,6 +23,26 @@
 DIRECTORY_DEFAULT = "."
 COPY_CODE_BUTTON_DEFAULT = False
 
+# These are the same colors used in pl-external-grader-result
+ansi_colors = {
```

Comment:
```suggestion
ANSI_COLORS = {
```
Convention for class constants is to have them in all caps.

Reply:
Done in 118f90313.